server/src/routes/sectionsRoute.ts

```diff
@@ -1,17 +1,18 @@
 import {
-  addSectionController,
-  deleteSectionController,
-  editSectionController,
-  getProjectSectionsController,
+  addSection,
+  deleteSection,
+  editSection,
+  getProjectSections,
 } from '../controllers';
 
-import { checkAuth } from '../middlewares';
+import { asyncMiddleware, checkAuth } from '../middlewares';
 
 const sectionsRoute = require('express').Router();
 
-sectionsRoute.post('/section', addSectionController);
-sectionsRoute.delete('/section/:id', deleteSectionController);
-sectionsRoute.put('/section/:id', editSectionController);
-sectionsRoute.get('/project/:id/sections', getProjectSectionsController);
+sectionsRoute.use(checkAuth);
```

Comment:
check auth also has try and catch, so we can wrap our middleware with async-middleware and remove try-catch block